build.gradle

```diff
@@ -0,0 +1,70 @@
+/**
+* The application plugin facilitates creating an executable JVM application.
+*/
+
+apply plugin: 'application'
+
+/**
+* This line specifies where the main.java file is found.
+*/
+
+mainClassName='View.Main'
+
+/**
+* This statement declares where Gradle is to find directories 
```

Comment:
This trailing white space needs to be removed.

Reply:
Deleted